### DISCUSSION TITLE
Brain ROI visualizations

### DISCUSSION THREAD

**Discussion by sanaashrafali** [now-i-know-my-abcd/docs#11] [Stats & Code Help] [answered]
What's the best way to visually represent correlations between brain (ROI) findings and clinical correlates when you want to highlight a couple key significant ones? (a heat map is not helpful in this case)

**Comment by sanaashrafali** [comment, marked as answer]
Below are options in both R and Python:

- R
 https://ggplot2.tidyverse.org/reference/geom_hex.html
 https://cran.r-project.org/web/packages/corrplot/vignettes/corrplot-intro.html
 https://github.com/ggseg/ggseg

- Python
 https://nilearn.github.io/plotting/index.html

**Reply by jarodroland** [reply]
Your Python link to Nilearn plotting is broken.
Try https://nilearn.github.io/dev/plotting/index.html or https://nilearn.github.io/stable/index.html

**Comment by JunhongYu** [comment]
i'm sharing some R markdown codes below that i've prepared for use in my lab. These codes display Desterieux (cortical thickness) and subcortical ROI data from the abcd_mrisdp10201.txt file. In order for these visualizations to work, I have matched the labels from abcd_mrisdp10201.txt to ones in the ggseg package.  Some regions from the abcd_mrisdp10201 file cannot be mapped on to the ggseg figures because of the way the brain is sliced up in the visualizations

https://rpubs.com/gerardyu/vizCT
https://rpubs.com/gerardyu/vizSubCort
